config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
+  
+  root 'restaurants#index'
+
   resources :restaurants
+
+  get 'restaurants/:id/destroy' => 'posts#destroy',  as: :delete_restaurant
```

Comment:
When I rake routes, just follow it’s path as ‘posts#destroy’ instead of ‘restaurants#destroy’ like I had, correct?

 

Why did it still appear to function?

 

Heidi

 

From: Brian Gates [mailto:notifications@github.com] 
Sent: Tuesday, June 16, 2015 1:55 AM
To: TheIronYard--Orlando/ror_granular_app
Cc: Heidi L. Roude
Subject: Re: [ror_granular_app] ahhhhh, the domino errors and subsequent headaches (#14)

 

In config/routes.rb <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14#discussion_r32492934> :

>    resources :restaurants
> +
> +  get 'restaurants/:id/destroy' => 'posts#destroy',  as: :delete_restaurant

a destroy route occurs as part of resources. You just need to change the link that's supposed to destroy a restaurant to use the proper method.

—
Reply to this email directly or view it on GitHub <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14/files#r32492934> .  <https://github.com/notifications/beacon/AL4F_PwQSjqx4R6YlfGlRJW51XJRn1Jqks5oT7GfgaJpZM4FD_cN.gif>